applications/annotate_points.md

```diff
@@ -0,0 +1,383 @@
+                df.loc[fname][scorer][label]['x'] = coord[2]
+                df.loc[fname][scorer][label]['y'] = coord[1]
+
+            # write the dataframe
+            df.to_csv(output_path)
+        
+```
+## point_annotator()
+We will create the GUI within a function called `point_annotator()`. Wrapping the GUI creation in the function allows us to  See below for the function definition.
+
+```python
+def point_annotator(
+	im_path: str,
+	labels: List[str],
+	output_path: str='anno.csv',
+	scorer: str='user'
+):
+    """Create a GUI for annotating points in a series of images.
+
+    Parameters
+    ----------
+    im_path : str
+        glob-like string for the images to be labeled.
+    labels : List[str]
+        list of the labels for each keypoint to be annotated (e.g., the body parts to be labeled).
+    output_path : str
+        path to the csv file to which the annotations will be saved
+    scorer : str
+        name of the person performing the annotation
+    """
+
+```
+
+## Loading the video
+First, we load the movie to be annotated. Since behavior movies can be quite long, we will use a lazy loading strategy (i.e., we will only load the frames as they are used). Using [`dask-image`](https://github.com/dask/dask-image), we can construct an object that we can pass to napari for lazy loading in just one line. For more explanation on using dask to lazily load images in napari, see [this](https://napari.org/tutorials/applications/dask) tutorial.
+
+```python
+stack = imread(im_path)
+```
+
+We can then start the viewer Note that we use the `napari.gui_qt()` context manager to start and manage Qt event loop. We must provide the `contrast_limits` and set `is_pyramid=False` to prevent the image stack from being evaluated (i.e., loaded) upon layer creation. All of the following GUI code should be within the `napari.gui_qt()` context manager.
+
+```python
+with napari.gui_qt():
+    viewer = napari.view_image(stack, contrast_limits=[0, 256])
+```
+
+## Annotating with points
+We will annotate the features of interest using points in a napari Points layer. Each feature will be given a different label so that we can track them across frames. To achieve this, we will store the label in the `Points.properties` property in the 'label' key. We will instantiate the `Points` layer without any points. Therefore, we will initialize `Points.properties` with the key we will use and an empty array (`properties={'label': np.empty(0)}`). We also initialize the layer with the valid label names using the `default_properties` kwarg (`default_properties = {'label': np.array(labels)}`).
+
+```python
+properties = {'label': np.array(labels)}
+```
+
+We then add the points layer to the viewer using the `viewer.add_points()` method. As discussed above, we will  To visualize the feature each points represent, we set the edge color as a color cycle mapped to the `label` property (`edge_color='label'`). 
```

Comment:
Looks like something go cut off at the end of this sentence too

Reply:
Fixed!